.gitattributes

```diff
@@ -0,0 +1,4 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text eol=lf
```

Comment:
The reason I did all is linux doesn't require .sh for executable scripts. 

The other reason is a personal preference that the repo uses 1 line ending for all files to make it simple to know what should be used.